### ISSUE TITLE
AP Biology imported exercises with images show both in-book and in-exercise versions

### ISSUE THREAD

**Issue by oscryan** [openstax/cnx-recipes#5431]
# Brief Description of Scenario

When an exercise in a feature is imported from Exercises, if the image is already marked up in the book, it also shows the copy uploaded to Exercises.
<img width="1081" alt="Screenshot 2024-01-27 at 1 06 05 PM" src="https://github.com/openstax/cnx-recipes/assets/1895432/8e48a144-d8ca-46a3-ae32-f68259c8602f">
 to the exercise.

# Pattern of Occurrence 

# Acceptance Criteria

  
# Sample Mark-Up
```
<!-- Please paste here if applicable. -->
```

# Location(s)
- **osbooks repo:** osbooks-biology-bundle
- **collection slug:** biology-ap-courses
- **ref/branch:**
- **chapter/section:** 
- **surrounding text:**

**Comment by lwtchu** [comment]
Semi related to #5430 - once we get injected exercises working with the feature boxes, we need to check the content for redundancy and make a decision. Alina and Theresa know about this

**Comment by openstaxalina** [comment]
Makes sense. Just noting for Future Me/Us that in this case the figure itself is written into the question stem - it's not using the `context-cnxfeature` tag to pull it in: 

https://exercises.openstax.org/exercise/11@14

![Screenshot 2024-02-09 at 3 10 03 PM](https://github.com/openstax/cnx-recipes/assets/7863733/f8d6ada3-9939-4525-8d7c-b431a01038b9)

I believe the content fix will be to remove the image from Exercises, and input the `figure` or `note` ID into the`context-cnxfeature`field

**Comment by lwtchu** [comment]
@openstaxalina i think with these notes we might have to try a different approach based on some things we are doing for other feature boxes. but we can talk about it at some point
